examples/chaplin-brunch/.gitignore

```diff
@@ -1,8 +1,28 @@
-bower_components/
-!bower_components/todomvc-common/
+# NPM packages.
```

Comment:
This comment seems like not necessary to me. We all know what `node_modules` are right?